### DISCUSSION TITLE
How Setting power plane for specific .exe file ?

### DISCUSSION THREAD

**Discussion by alex3xelax** [winsiderss/systeminformer#1999] [Questions / Answers]
hi, i like setting hight priority for demanding game, is there also the ability to **setting specific  windows power plane** (balanced, performance ecc...) for a specific game exutable files ?

**Comment by Advait251206** [comment]
As far as I know, System Informer can save and reapply things like process priority, affinity, and certain process-specific settings, but it does **not** provide a built-in feature to automatically switch the Windows power plan based on a specific executable.

Windows power plans (Balanced, High Performance, Power Saver, Ultimate Performance, etc.) are generally system-wide settings rather than per-process settings.

A few alternatives you could consider:

### Option 1: Use a script

You can create a script that:

1. Switches to a High Performance power plan.
2. Launches the game.
3. Waits for the game to exit.
4. Restores your previous power plan.

For example:

```bat
@echo off
powercfg /setactive <HIGH_PERFORMANCE_GUID>
start /wait "" "C:\Games\MyGame.exe"
powercfg /setactive <BALANCED_GUID>
```

You can obtain the GUIDs with:

```cmd
powercfg /list
```

### Option 2: Use a third-party launcher

Some gaming tools and launchers can execute commands before and after starting a game, allowing you to switch power plans automatically.

### Option 3: Use Windows Game Mode

For many systems, enabling **Game Mode** and using the **High Performance** or **Ultimate Performance** plan globally may provide most of the benefits without needing per-game switching.

### Option 4: Process priority only

If your goal is simply to give a game more CPU time, setting a higher process priority in System Informer may be sufficient without changing the system power plan.

So, unless I've missed a recent feature, I don't believe System Informer currently supports assigning a specific Windows power plan directly to an executable and automatically switching plans when that executable starts.
